data/src/main/scala/cats/data/Validated.scala

```diff
@@ -0,0 +1,143 @@
+package cats
+package data
+
+import cats.data.Or.{LeftOr, RightOr}
+import cats.data.Validated.{Valid, Invalid}
+
+sealed abstract class Validated[+E, +A] extends Serializable {
```

Comment:
I don't doubt that you are right. I left It off because I didn't know what it bought us. What does it buy us?